sklearn_numba_dpex/tests/test_kmeans.py

```diff
@@ -6,9 +8,14 @@
 from sklearn.utils._testing import assert_allclose
 
 
+def test_placeholder():
+    """Placeholder test for CI"""
```

Comment:
maybe the following first test that checks that the main packages are installed:

```
try:
    import numba_dpex as dpex
except ImportError:
    import numba_dppy as dpex

import dpctl

dpctl.get_devices()

```
?

Reply:
Yes, let's add those imports for this test. :+1:

On a side-note, you should be able to use GitHub suggestions. For instance:

```suggestion
    """Placeholder test for CI"""
    try:
	    import numba_dpex as dpex
	except ImportError:
	    import numba_dppy as dpex
	
	import dpctl
	
	# There must be at least one usable device.
	assert len(dpctl.get_devices()) > 0
```